GettingStartedwithAnsibleusingGNS3.md

```diff
@@ -0,0 +1,283 @@
+# EXOS: Getting Started with Ansible using GNS3
+
+## Sections:
+* [Pre-requisites](#pre-requisites)
+* [Setting up GNS3 with EXOS-VM](#setting-up-gns3-with-exos-vm)
+* [Setting up GNS3 with Network Automation Docker Container](#setting-up-gns3-with-network-automation-docker-container)
+* [Setting up a Test Topology](#setting-up-a-test-topology)
+	- [EXOS Device Setup](#exos-device-setup)
+	- [Network Automation Container Setup](#network-automation-container-setup)
+* [System Setup](#system-setup)
+	- [Installing Ansible](#installing-ansible)
+	- [Setting up the Virtual Environment](#setting-up-the-virtual-environment)
+	- [Configuring the Local hosts file](#configuring-the-local-hosts-file)
+	- [Creating an Ansible configuration file](#creating-an-ansible-configuration-file)
+	- [Creating and configuring the Ansible inventory file](#creating-and-configuring-the-ansible-inventory-file)
+	- [Creating and configuring the Variable file](#creating-and-configuring-the-variable-file)
+* [Testing the Setup](#testing-the-setup)
+
+
+## Pre-requisites
+
+- GNS3 installed on any supported OS
+- GNS3 VM installed on any supported Hypervisor
+- Virtual EXOS Image
+
+## Setting up GNS3 with EXOS-VM
+
+1. Download the EXOS appliance template from [GNS3 Marketplace]
+2. Download the required qcow2 file from our [GitHub Website]
+3. Import the downloaded template in GNS3 and load the qcow2 file downloaded in step 2. Visit this [link] for details
+4. Drag the installed appliance to the workspace and start it up
+5. Run few show commands on the device to make sure that its working fine
```

Comment:
I would slightly reword this. Maybe say "Run some show commands to make sure it is working. For example `show switch` and `show version`.

Reply:
done